.sync/containers/Ubuntu-22/Dockerfile

```diff
@@ -77,113 +78,141 @@ RUN apt-get update && \
         gcc-${GCC_MAJOR_VERSION}-aarch64-linux-gnu \
         gcc-${GCC_MAJOR_VERSION}-riscv64-linux-gnu \
         gcc-${GCC_MAJOR_VERSION}-arm-linux-gnueabi \
-        gcc-${GCC_MAJOR_VERSION}-arm-linux-gnueabihf \
-        && \
-    curl -O https://bootstrap.pypa.io/get-pip.py && \
-    {% endraw %}python{{ sync_version.python_version }}{% raw %} get-pip.py && \
-    {% endraw %}python{{ sync_version.python_version }}{% raw %} -m pip install --upgrade setuptools && \
-    rm get-pip.py && \
-    apt-get upgrade -y && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
-
-ENV GCC5_AARCH64_PREFIX /usr/bin/aarch64-linux-gnu-
-ENV GCC5_ARM_PREFIX     /usr/bin/arm-linux-gnueabi-
-ENV GCC5_RISCV64_PREFIX /usr/bin/riscv64-linux-gnu-
-
-RUN pip install pip lcov_cobertura --upgrade
-
-# RUN npm install -g npm
-RUN npm install -g \
+        gcc-${GCC_MAJOR_VERSION}-arm-linux-gnueabihf && \

```

Comment:
looks like a double `&&` here between line 81 and 82?

```
gcc-${GCC_MAJOR_VERSION}-arm-linux-gnueabihf && \
    && curl -O https://bootstrap.pypa.io/get-pip.py \
```

Reply:
ops, I see, my bad